packages/paste-website/src/pages/patterns/filter/index.mdx

```diff
@@ -81,12 +81,11 @@ This pattern provides recommendations for selection methods, layout, and behavio
     <UnorderedList>
       <ListItem>List all filtering criteria available.</ListItem>
       <ListItem>Map out the relationships between different filters and choose the ones that will be part of your feature. Remember filters should give users a sense of control and not overwhelm them.</ListItem>
-      <Blockquote
-        name="Defining Helpful Filter Categories and Values for Better UX"
-        source="Nielsen Norman Group"
-        sourceUrl="https://www.nngroup.com/articles/filter-categories-values/"
-      >
-        A truly usable faceted search provides filter categories and filter values that are appropriate, predictable, free of jargon, and prioritized
+      <Blockquote url="https://www.nngroup.com/articles/filter-categories-values/">
+        <BlockquoteContent>
+              A truly usable faceted search provides filter categories and filter values that are appropriate, predictable, free of jargon, and prioritized
```

Comment:
```suggestion
              A truly usable faceted search provides filter categories and filter values that are appropriate, predictable, free of jargon, and prioritized.
```

nit: let's just make this change if any other changes are needed - not necessary to do another commit just for this